code/obj/item/clothing/armor.dm

```diff
@@ -9,6 +9,7 @@
 	icon_state = "armor"
 	item_state = "armor"
 	body_parts_covered = TORSO|LEGS|ARMS
+	hides_from_examine = C_UNIFORM|C_GLOVES|C_SHOES|C_BELT
```

Comment:
This is probably a bad idea since a lot of stuff is a subtype of this.

Reply:
oh, i know but considering most armor already covers all this it seemed best to just change all the edge cases instead of adding it to every subtype